extensions/cornerstone/src/components/StudySummaryFromMetadata.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { DicomMetadataStore, utils } from '@ohif/core';
+import { StudySummary } from '@ohif/ui-next';
+
+const { formatDate } = utils;
+
+export function StudySummaryFromMetadata({ studyInstanceUID }) {
```

Comment:
Can we use dcmjs style namings? so `StudyInstanceUID` for the prop? below you are actually doing `const { StudyDate, StudyDescription } = instanceMeta;`